agent/api/container/container.go

```diff
@@ -272,6 +274,19 @@ type MountPoint struct {
 	ReadOnly      bool   `json:"readOnly"`
 }
 
+// Log Router things
```

Comment:
See https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences, comments should begin with the name of the thing being described and end in a period. Maybe "// LogRouter contains information needed by a log router container."